src/platform/user/profile/vap-svc/constants/addressValidationMessages.js

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable react/prop-types */
```

Comment:
So I'm having issues setting any prop types within this particular object. You can see that I attempted to set some prop types at the end of the file, but they are not detected through eslint as actually being set... so that is strange. Probably because of the constant based property access is my guess.